Src/Base/AMReX_FArrayBox.H

```diff
@@ -346,7 +380,7 @@ public:
     * It must be the case the comp >= 0 && num_comp >= 1 &&
     * (comp+num_comp) <= nComp().  The FAB is written out in
     * whatever format you've set, with the default format being
-    * ASCII.  The FAB that is written to disk will be an
+    * NATIVE.  The FAB that is written to disk will be an
     * num_comp component FAB.
```

Comment:
The above overload of this function should also have a default on NATIVE, not ASCII.

Reply:
I mean that the comment should be updated, the code already uses NATIVE

Reply:
That's right.  Fixed.